view_all_set.php

```diff
@@ -122,6 +122,7 @@
 			exit; # stop here
 		}
 	}
+	$t_setting_arr = filter_ensure_valid_filter( $t_setting_arr );
```

Comment:
Should we ensure value in the else for `if( false === $t_setting_arr )`?  In this place, if it is value, calling the ensure valid header doesn't make sense.